### PR TITLE
Add UI login and register

### LINKED ISSUE #7: Add UI login/register/logout

### PR DESCRIPTION
**Pull request by gilly7**
Added UI login and register UI

### CODE REVIEW

File: frontend/app/layout.tsx [hunk 1]
```diff
@@ -14,6 +14,7 @@ import { ThemeProvider } from "@mui/material/styles";
 import Link from "next/link";
 import React from "react";
 import theme from "./theme";
+import AppHeader from "./AppHeader";
```
Comment (etowett) [inline frontend/app/layout.tsx:17]: Where is this file? Looks like you are importing a non existent file?

**Review by etowett — CHANGES_REQUESTED** [review]

File: frontend/app/layout.tsx (hunk 1 above)
Comment (gilly7) [reply frontend/app/layout.tsx:17]: Pls check latest commit

File: frontend/app/AppHeader.tsx [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+  return (
+    <AppBar position="static" color="primary" elevation={0}>
+      <Toolbar>
+        <Typography variant="h6" component="div" sx={{ flexGrow: 1 }}>
+          <Link
+            href="/"
+            passHref
+            style={{ color: "inherit", textDecoration: "none" }}
+          >
+            DuBlog
```
Comment (etowett) [inline frontend/app/AppHeader.tsx]: We probably want to rename this?

File: frontend/app/AppHeader.tsx [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+            </Button>
+            <Button color="inherit" component={Link} href="/register">
+              Register
+            </Button>
+          </>
+        )}
+      </Toolbar>
+    </AppBar>
+  );
+}
```
Comment (etowett) [inline frontend/app/AppHeader.tsx:61]: Add line break?

**Review by etowett — APPROVED** [review]
Good stuff, please check the comments

File: frontend/app/AppHeader.tsx (hunk 3 above)
Comment (gilly7) [reply frontend/app/AppHeader.tsx:61]: Thank you for the suggestion.

To add a line break between the "Login" and "Register" buttons, we can wrap each button in a Box component from MUI with some margin.

File: frontend/app/AppHeader.tsx (hunk 2 above)
Comment (gilly7) [reply frontend/app/AppHeader.tsx]: Renamed.

File: frontend/app/AppHeader.tsx [hunk 4]
```diff
@@ -2,10 +2,11 @@
 
 import React from "react";
 import AppBar from "@mui/material/AppBar";
-import Toolbar from "@mui/material/Toolbar";
+import Toolbar from "@mui/material/Toolbox";
```
Comment (etowett) [inline frontend/app/AppHeader.tsx:5]: Any reason for this rename?

**Pull request merged**